src/add/operator/zip-static.ts

```diff
@@ -1,5 +1,5 @@
 import {Observable} from '../../Observable';
 import {zip} from '../../operator/zip-static';
-Observable.zip = zip;
+Observable.zip = <any>zip;
```

Comment:
If this change is really needed (the build breaks without it), I think we should integrate this PR with this change and I can modify the script afterwards to handle this case. But I'd drop it if this is not the case.